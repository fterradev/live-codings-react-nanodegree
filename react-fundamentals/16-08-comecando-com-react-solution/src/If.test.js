import React from 'react';
import { shallow } from 'enzyme';
import If from './If';

it('renderiza corretamente', () => {
  const test = (
    <If test={true}>
      <span>Olá, teste</span>
    </If>
  )
  expect(shallow(test)).toMatchSnapshot()
})

it('renderiza corretamente sem elementos filhos', () => {
  const test = (
    <If test={false}>
      <span>Olá, teste</span>
    </If>
  )
  expect(shallow(test)).toMatchSnapshot()
})