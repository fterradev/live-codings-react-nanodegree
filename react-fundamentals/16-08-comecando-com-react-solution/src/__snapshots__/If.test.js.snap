// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza corretamente 1`] = `
<span>
  Ol√°, teste
</span>
`;

exports[`renderiza corretamente sem elementos filhos 1`] = `null`;
